docs/networks/terraform-and-ansible.md

```diff
@@ -4,6 +4,8 @@ order: 3
 
 # Terraform & Ansible
 
+> Note: These commands/files are not being maintained by the tendermint team currently. Please use them carefully.
```

Comment:
added this section for now. It is fine to have them in the repo for now. we can come back in the future if anything changes @melekes